.github/workflows/codeql-query.yml

```diff
@@ -122,8 +121,6 @@ jobs:
 
       - name: Run query
         uses: ./query
-        env:
```

Comment:
Ok, unfortunately it does not work. See https://github.com/dsp-testing/qc-controller/actions/runs/4981941371

Seems it's the `@octokit/auth-action` dependency requiring that, even though we end up not using it and overwriting the authorization header with the signed auth token. We might be able to hack around that, but it's not essential and probably not worth it in this PR. What do you think?

Reply:
Thanks for looking at that before I got to it - saved me some time. 

Yeah I think we can take that as tech debt for now.